src/main/java/io/bonitoo/qa/VirDevRuntimeException.java

```diff
@@ -0,0 +1,10 @@
+package io.bonitoo.qa;
+
+/**
+ * Encapsulates Virtual Device exceptions that occur during runtime.
+ */
+public class VirDevRuntimeException extends RuntimeException {
```

Comment:
VirtualDeviceRuntimeException instead of VirDev better explains what it is

Reply:
I thought a name nearly 32 characters long was a bit much and my old C++ habits prodded me to shorten it.  Actually it's only 30 characters long.  Renamed.